examples/pennylane_skip_pytorch_noise.py

```diff
@@ -0,0 +1,205 @@
+
+* `PennyLane-Forest plugin <https://github.com/rigetti/pennylane-forest>`_, in order
+  to access the QVM as a PennyLane device. This can be installed via pip:
+
+  .. code-block:: bash
+
+      pip install pennylane-forest
+
+* `PyTorch <https://pytorch.org/get-started/locally/>`_, in order to access the PyTorch
+  QNode interface. Follow the link for instructions on the best way to install PyTorch
+  for your system.
+"""
+
+##############################################################################
+# Setting up the device
+# ---------------------
+#
+# Once the dependencies above are installed, let's begin importing the required packages
+# and setting up our quantum device.
+#
+# To start with, we import PennyLane, and, as we are using the PyTorch interface,
+# PyTorch as well:
+
+import pennylane as qml
+import torch
+from torch.autograd import Variable
+
+##############################################################################
+# Note that we do not need to import the wrapped version of NumPy provided by PennyLane,
+# as we are not using the default QNode NumPy interface. If NumPy is needed, it is fine to
+# import vanilla NumPy for use with PyTorch and TensorFlow.
+#
+# Next, we will create our device:
+
+dev = qml.device("forest.qvm", device="2q", noisy=True)
+
+##############################################################################
+# Here, we create a noisy two-qubit system, simulated via the QVM. If we wish, we could
+# also build the model on a physical device, such as the ``Aspen-1`` QPU.
+
+
+##############################################################################
+# Constructing the QNode
+# ----------------------
+#
+# Now that we have initialized the device, we can construct our quantum node. Like the
+# other tutorials, we use the :mod:`qnode decorator <pennylane.decorator>` to convert
+# our quantum function (encoded by the circuit above) into a quantum node
+# running on the QVM.
+
+
+@qml.qnode(dev, interface="torch")
+def circuit(phi, theta):
+    qml.RX(theta, wires=0)
+    qml.RZ(phi, wires=0)
+    return qml.expval(qml.PauliZ(0))
+
+
+##############################################################################
+# To make the QNode 'PyTorch aware', we need to specify that the QNode interfaces
+# with PyTorch. This is done by passing the ``interface='torch'`` keyword argument.
+#
+# As a result, this QNode will be set up to accept and return PyTorch tensors, and will
+# also automatically calculate any analytic gradients when PyTorch performs backpropagation.
+
+
+##############################################################################
+# Optimization
+# ------------
+#
+# We can now create our optimization cost function. To introduce some additional
+# complexity into the system, rather than simply training the variational circuit
+# to 'flip a qubit' from state :math:`\ket{0}` to state :math:`\ket{1}`, let's also
```

Comment:
Looking at the jupyter notebook again, it _seems_ to be able to render math, but not to be able to pick up our macros. It would be worthwhile to hard-code the math in these examples (shouldn't be too many occasions)

Reply:
:+1: I agree this would be best. Jupyter notebooks without additional configuration only support default MathJaX macros.

Reply:
All custom macros have been removed.